.github/workflows/build.yml

```diff
@@ -1054,6 +1055,24 @@ jobs:
           docker load --input ./osrd-front-nginx.tar
           docker load --input ./osrd-osrdyne.tar
 
+      - name: Detect Playwright version
+        id: detect_playwright_version
+        run: |
+          PLAYWRIGHT_VERSION=$(cd front && yarn list --pattern playwright --json | jq -r '.data.trees[].name | split("@")[-1]' | sort -u)
+          if [ "$(echo "$PLAYWRIGHT_VERSION" | wc -l)" -ne 1 ]; then
+              echo "Error: Zero or multiple playwright versions found: $PLAYWRIGHT_VERSION" >&2
+              exit 1
+          fi
+          echo "Detected Playwright version: $PLAYWRIGHT_VERSION"
+          echo "PLAYWRIGHT_VERSION=$PLAYWRIGHT_VERSION" >> $GITHUB_ENV
+
+      - name: Build Playwright container
+        run: |
+          docker build --build-arg PLAYWRIGHT_VERSION=v$PLAYWRIGHT_VERSION \
+            --build-arg FRONT_TESTS_IMAGE=${{ fromJSON(needs.build.outputs.stable_tags).front-tests }} \
+            -t osrd-playwright:latest \
+            -f front/docker/Dockerfile.playwright-ci .
```

Comment:
If if the goal is to speed up the ci, we probably shouldn't transfer the whole repo into the context (since the dockerfile doesn't use it anyways) Something like `docker build -t osrd-playwright [etc] - <front/docker/Dockerfile.playwright-ci

Reply:
Done thanks!